R/custom_disconnect_message.R

```diff
@@ -50,12 +50,14 @@
 #' )
 custom_disconnect_message <- function(
     refresh = "Refresh page",
+    reset = "Reset page",
     dashboard_title = NULL,
     links = NULL,
     publication_name = NULL,
     publication_link = NULL,
     support_contact = "explore.statistics@education.gov.uk",
-    custom_refresh = NULL) {
+    custom_refresh = NULL,
+    custom_reset = NULL) {
   # Check links are valid
   is_valid_sites_list <- function(sites) {
```

Comment:
Please can you switch this validation to use `validate_dashboard_url()` from R/validation.R and remove `is_valid_sites_list()` from this script.